src/ontology/mondo-ingest.Makefile

```diff
@@ -437,6 +437,7 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db $(MAPPINGSDIR)/rejected-mappings.tsv
 	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
 	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical-2.sssom.tsv
+	pip install bioregistry
```

Comment:
Hmm, missing bioregistry, or a specific version of it, I see.
```suggestion
	pip-bioregistry
```

I suggest this because I take it that it's not simply that `bioregistry` is missing, but maybe you want the latest version of it? In which case you want:
```make
pip-%:
	 python3 -m pip install --upgrade $*
```

@twhetzel The reason we have some `pip` install commands scattered throughout the makefile, and why they are for specific packages, rather than doing the a full install (e.g. `pip install -r python-requirements-unlocked.txt` or `poetry install`), is because for the particular piece of work we're working on, there's sometimes a new upgraded version of that package that has not yet made its way into the versioning for the latest release ODK yet. This is also partially alternatively solved by using the ODK dev container, which is what I think we're pretty much always using now.

@matentzn Are we using the ODK dev container for builds too now? I haven't seen an ODK release since September.